tests/jballerina-unit-test/src/test/resources/test-src/statements/ifelse/if-stmt.bal

```diff
@@ -217,6 +217,46 @@ function testResetTypeNarrowingWithBlockStmt() {
     }
 }
 
+type Type A|Tuple|List;
+type A "A";
+type Tuple ["tuple", Type, Type...];
+type List ["list", int?, Type];
+
+function testCustomCircularTupleTypeWithIsCheck() {
```

Comment:
These are rather type narrowing tests. Shall we move them to https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-unit-test/src/test/resources/test-src/statements/ifelse/type-guard.bal

Reply:
Updated. 9db7b4456e47d7a3eccf59d70f8d182750da400f